docs/install/2-install-server-with-docker.rst

```diff
@@ -305,11 +296,18 @@ There are two ways you can achieve this:
             image: containers.inmanta.com/containers/service-orchestrator:|version_major|
             ports:
                 - 8888:8888
+            env_file: ./resources/my-env-file
+            environment:
+                INMANTA_DATABASE_HOST: inmanta_db
+                INMANTA_DATABASE_USERNAME: inmanta
+                INMANTA_DATABASE_PASSWORD: inmanta
+                INMANTA_CONFIG_AGENT_DEPLOY_INTERVAL: 800
+                INMANTA_CONFIG_AGENT_REPAIR_INTERVAL: 86400
+                INMANTA_SERVER_BIND_PORT: 8888
```

Comment:
* The log rotation section talks about the logs of the SSH server. That's not required anymore, because the container doesn't have an SSH server anymore.
* The handling of the `INMANTA_UID` and `INMANTA_GID` environment variables is also something that was done by the old entrypoint, right?

Reply:
I'm not quite sure about the UID and GID. I never had to deal with them when deploying the orchestrator with container.

Reply:
After some more testing, uid and gid env do not work anymore.